pennylane/data/data_manager.py

```diff
@@ -162,12 +197,12 @@ def _generate_folders(node, folders):
 
 
 def load(
```

Comment:
Looking at this example, 

```
H2_dataset = qml.data.load(data_type='qchem',molname='H2', basis='6-31G', bondlength='0.46')
print(H2_dataset)
[<pennylane.data.dataset.Dataset object at 0x7f14e4369640>]
```

the returned object is a list of length one. Why it is a list and not just the datasets object?

Reply:
I would prefer to keep all the return types consistent. Otherwise, the user would have to check whether the returned type is a list or a single object, which I believe is very inconvenient.

Reply:
discussed this with Diego and Utkarsh. I'd rather the return type be consistent (`list[Dataset]`), which will be especially reliable for writing some automation/script that uses this library. Otherwise, a user's script would have to check if the return type is a Dataset or a list. I can just return the object if you feel strongly against this.

Reply:
Just curious, why the returned object needs to be a list? Is there any scenario that the list contains more than one dataset? (I think there is something about consistency that I am not  aware of.)

Reply:
any time a user provides more than one value for a parameter. Example with 2 bondlengths:
```pycon
>>> from pennylane.data import load
>>> load('qchem', molname='H2', basis='STO-3G', bondlength=[1, 0.75])
[<pennylane.data.dataset.Dataset object at 0x147f43d90>, <pennylane.data.dataset.Dataset object at 0x104c3b430>]
>>> for d in _:
...     print(d._folder)
... 
datasets/qchem/H2/STO-3G/0.75
datasets/qchem/H2/STO-3G/1.0

Reply:
OK, thanks.